packages/app/src/components/DBRowOverviewPanel.tsx

```diff
@@ -49,17 +59,23 @@ export function RowOverviewPanel({
   );
 
   const isHttpRequest = useMemo(() => {
-    return eventAttributes?.['http.url'] != null;
-  }, [eventAttributes]);
+    const attributes =
+      eventAttributesExpr && dataAttributes?.[eventAttributesExpr];
+    return attributes?.['http.url'] != null;
```

Comment:
I want to point out this is trace specific attribute (https://github.com/open-telemetry/opentelemetry-js/blob/eaebf765d8408b29264af5c50e51ce494781455e/semantic-conventions/src/trace/SemanticAttributes.ts#L91).

I'd suggest to import these attributes from semantic convention library

Reply:
not a blocker. something we can clean it up later